UPSTREAM_CONFIG.md

```
@@ -0,0 +1,170 @@
+提示：如果某个下游企业未在 `corp_list` 中出现，通常是该企业还未确认应用共享或共享未生效。
+
+## 工作原理
+
+1. 当收到消息时，插件检测消息中的 `ToUserName`（CorpID）
+2. 如果 `ToUserName` 与主 CorpID 不同，则识别为上下游用户
+3. 回复时使用 `wecom-agent-upstream:{accountId}:{corpId}:{userId}` 格式的 target
+4. Outbound 模块解析该 target，使用对应的上下游 Agent 配置发送消息
+
+## 日志示例
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)